src/components/Newsletter/Newsletter.js

```diff
@@ -35,50 +42,131 @@ class Newsletter extends React.Component {
       });
   }
 
+  handleSelectChange = selectedOption => {
+    this.alertsSelectionArray = selectedOption;
+    this.setState({
+      alertSignUps: this.alertsSelectionArray
+        .map(item => item.full_fips_code)
+        .join(','),
+    });
+  };
+
   render() {
     const { stateId, county } = this.props;
+
+    const stateDataset = US_STATE_DATASET.state_dataset.map(state => {
+      return {
+        ...state,
+        full_fips_code: state.state_fips_code + '000',
```

Comment:
FWIW, adding '000' seems nonstandard.  The FIPS code for Washington is just `53`, not `53000` and this is what we use e.g. in our [ingested data sets](https://github.com/covid-projections/covid-data-public/blob/master/data/test-and-trace/state_data.csv).  It maybe doesn't matter _too_ much as long as we know this is what we're using and are consistent everywhere, but it could be a surprise / cause bugs if somebody new comes to this code and assumes we're using normal state FIPS codes.

Since it would simplify our code to _not_ add '000', it seems strictly better to use the raw state FIPS code.

Reply:
cool fixed